### DISCUSSION TITLE
sfserver needs to be updated

### DISCUSSION THREAD

**Discussion by kriegtiger** [GameServerManagers/LinuxGSM#4401] [General]
It's a needed fix but not a 'bug' per se so I am posting here. 

Recently tried to play my satisfactory game after a long time off and ran all the updates available (lgsm, game server files, steamcmd). Ran './sfserver start' and it failed with the following message.

![image](https://github.com/GameServerManagers/LinuxGSM/assets/6956044/515c9331-8419-467c-a477-663db41cb95f)

Went looking through that path and there were no more UE4 executables. I also found that the libsteam_api.so link was broken:
![image](https://github.com/GameServerManagers/LinuxGSM/assets/6956044/4998267a-8566-45d9-994e-2b864e0c74ff)

Deleted the broken link and recreated it pointing to the proper spot, and created a link for sfserver so the missing reference is present and goes to the current executable. 
![image](https://github.com/GameServerManagers/LinuxGSM/assets/6956044/7adfadef-e4df-4818-8788-346cd52d7013)

Game launches and works great after that.

**Comment by SaengerItsWar** [comment]
did you try to update the the LGSM Scripts with `./sfservewr update-lgsm` cause there were an update to the [_default.cfg](https://github.com/GameServerManagers/LinuxGSM/blob/master/lgsm/config-default/config-lgsm/sfserver/_default.cfg#L160) a month ago that updated the path to the binary
